extensions/amp-animation/0.1/css-expr-ast.js

```diff
@@ -596,6 +601,111 @@ export class CssTranslateNode extends CssFuncNode {
 
 
 /**
+ * An AMP-specific `width()` and `height()` functions.
+ */
+export class CssDimSizeNode extends CssNode {
+  /**
+   * @param {string} dim
+   * @param {?string=} opt_selector
+   * @param {?string=} opt_selectionMethod Either `undefined` or "closest".
+   */
+  constructor(dim, opt_selector, opt_selectionMethod) {
+    super();
+    /** @const @private */
+    this.dim_ = dim;
+    /** @const @private */
+    this.selector_ = opt_selector || null;
+    /** @const @private */
+    this.selectionMethod_ = opt_selectionMethod || null;
+  }
+
+  /** @override */
+  css() {
+    // No CSS represention.
+    return '';
```

Comment:
Can we log this? This can lead to subtle bugs if this gets called expecting a valid result state.